R/read_scenario_data.R

```diff
@@ -0,0 +1,17 @@
+#' Read in scenario data
+#'
+#' Function reads in scenario data and checks for existence of required columns.
+#'
+#' @param path Path to dir holding scenario data.
+#'
+#' @return A tibble holding scenario data.
+read_scenario_data <- function(path) {
+  scenario_data <- readr::read_csv(path, col_types = "ccccccncn")
```

Comment:
let's make the col_types explicit while we are at it

Reply:
what do you mean by that?